mantle/cmd/kola/testiso.go

```diff
@@ -123,17 +123,33 @@ func newBaseQemuBuilder() *platform.QemuBuilder {
 	return builder
 }
 
-func newQemuBuilder() *platform.QemuBuilder {
+func newQemuBuilder(isPXE bool) *platform.QemuBuilder {
 	builder := newBaseQemuBuilder()
 	sectorSize := 0
 	if kola.QEMUOptions.Native4k {
 		sectorSize = 4096
 	}
-	builder.AddPrimaryDisk(&platform.Disk{
-		Size: "12G", // Arbitrary
+	primaryDiskAdded := false
+	if system.RpmArch() == "s390x" {
+		// For s390x PXE installs the network device has the bootindex of 1.
+		// Do not use a primary disk in case of net-booting for this test
+		if isPXE {
```

Comment:
Minor, but why not compress to:
`if system.RpmArch() == "s390x" && isPXE {`
?

Reply:
yes..done.